modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,499 @@
+//module included in the following assemblies:
+//
+//networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-troubleshooting.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="metrics-commands-anp_{context}"]
+== Using nbctl commands for ANP and BANP
+
+To troubleshoot an unsuccessful setup, start by looking at OVN Northbound database (nbdb) objects which include the `ACL` and `AdressSet`. To view thee nbdb, you need to be inside the pod on that node to see what objects are in the node.
```

Comment:
Shouldn't we include also how to get the port groups, as in the [upstream docs](https://github.com/ovn-org/ovn-kubernetes/pull/4431)?

cc @tssurya

Reply:
yes we must